docs/user-guide/petset/bootstrapping/index.md

```diff
@@ -139,7 +141,8 @@ The role of the peer finder:
 You can invoke the peer finder inside the Pets we created in the last example:
 
 ```shell
-$ kubectl exec -it vm-0 /bin/bash
```

Comment:
yeah it's fine eitherway so lets get this in. We're not consistent about bash vs sh. I use bash in petset examples because the init-container startup-scripts use bash syntax.